cg/cli/deliver/base.py

```diff
@@ -167,7 +167,7 @@ def deliver_ticket(
     cg_context: CGConfig = context.obj
     deliver_ticket_api = DeliverTicketAPI(config=cg_context)
     is_upload_needed = deliver_ticket_api.check_if_upload_is_needed(ticket=ticket)
-    if is_upload_needed:
+    if is_upload_needed or force_all:
```

Comment:
Should the command help be updated to reflect this change?

Reply:
Agreed - fixing